README.md

```diff
@@ -56,6 +64,8 @@ Run a specific `gohack` version:
 $ GO111MODULE=off go get -u github.com/myitcv/gobin
 ```
 
+or grab a binary from [the latest release](https://github.com/myitcv/gobin/releases/latest).
```

Comment:
I'd personally link to the releases page - the latest one is obvious, and one can see the changelog leading up to it.